unity-renderer/Assets/Rendering/ProceduralSkybox/ToolProceduralSkybox/Scripts/Editor/Procedural Skybox/SkyboxEditor.cs

```diff
@@ -0,0 +1,1344 @@
+
+            SkyboxEditorWindow window = (SkyboxEditorWindow)EditorWindow.GetWindow(typeof(SkyboxEditorWindow));
+            //SkyboxEditorWindow.selectedConfiguration = new SkyboxConfiguration();
+            window.minSize = new Vector2(500, 500);
+            window.Show();
+            window.InitializeWindow();
+        }
+
+        public void InitializeWindow()
+        {
+            if (Application.isPlaying)
+            {
+                if (SkyboxController.i != null)
+                {
+                    isPaused = SkyboxController.i.IsPaused();
+                }
+
+            }
+            EnsureDependencies();
+        }
+
+        void OnGUI()
+        {
+            GUILayout.BeginArea(new Rect(10, 0, position.width - 20, position.height));
+
+            GUILayout.Space(32);
+            RenderConfigurations();
+            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider);
+
+            GUILayout.Space(32);
+            RenderTimePanel();
+            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider);
+            GUILayout.Space(12);
+
+            panelScrollPos = EditorGUILayout.BeginScrollView(panelScrollPos, GUILayout.Width(position.width - 20), GUILayout.Height(position.height - 90));
+            GUILayout.Space(32);
+            showBackgroundLayer = EditorGUILayout.Foldout(showBackgroundLayer, "BG Layer", true);
+            if (showBackgroundLayer)
+            {
+                EditorGUI.indentLevel++;
+                RenderBackgroundColorLayer();
+                EditorGUI.indentLevel--;
+            }
+            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider);
+
+            GUILayout.Space(32);
+            showAmbienLayer = EditorGUILayout.Foldout(showAmbienLayer, "Ambient Layer", true);
+            if (showAmbienLayer)
+            {
+                EditorGUI.indentLevel++;
+                RenderAmbientLayer();
+                EditorGUI.indentLevel--;
+            }
+            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider);
+
+            GUILayout.Space(32);
+            showFogLayer = EditorGUILayout.Foldout(showFogLayer, "Fog Layer", true);
+            if (showFogLayer)
+            {
+                EditorGUI.indentLevel++;
+                RenderFogLayer();
+                EditorGUI.indentLevel--;
+            }
+            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider);
+
+            GUILayout.Space(32);
+            showDLLayer = EditorGUILayout.Foldout(showDLLayer, "Directional Light Layer", true);
+            if (showDLLayer)
+            {
+                EditorGUI.indentLevel++;
+                RenderDirectionalLightLayer();
+                EditorGUI.indentLevel--;
+            }
+            EditorGUILayout.LabelField("", GUI.skin.horizontalSlider);
+
+            GUILayout.Space(32);
+
+            // Render Slots
+            RenderSlots();
+
+            // Render Texture Layers
+            //RenderTextureLayers();
```

Comment:
If `RenderTextureLayers` is not needed, we should remove the method altogether.

Reply:
Done. Making all these changes in a new commit